src/main/java/com/commercetools/sync/inventories/helpers/InventorySyncStatistics.java

```diff
@@ -0,0 +1,32 @@
+package com.commercetools.sync.inventories.helpers;
+
+import com.commercetools.sync.commons.helpers.BaseSyncStatistics;
+
+import static java.lang.String.format;
+
+/**
+ * Statistics of inventory synchronisation process.
```

Comment:
this comment does not introduce any new information
class's name is sufficiently self-descriptive